python/lsst/sims/maf/slicers/userPointsSlicer.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, ra, dec, verbose=True, lonCol='fieldRA', latCol='fieldDec',
         if len(ra) != len(dec):
             raise ValueError('RA and Dec must be the same length')
         self.nslice = np.size(ra)
+        self.ra = ra
+        self.dec = dec
```

Comment:
Actually - can you back out this change? I fixed it in the PR I just submitted by changing the _eq_ comparison. None of the other slicers are currently keeping a ra/dec attribute, but rather using the slicePoints['ra'] and ['dec'] dictionaries, and keeping these consistent seems likely to help us with maintainability in the future.